test/integration/integration_test_fixtures.py

```diff
@@ -21,6 +19,15 @@
 
 
 def determineIpVersionsFromEnvironment():
+  """Determine the IP version(s) for test execution from the environment.
+
+  Raises:
+      NighthawkException: raised when no ip version could be determined, or
+      an invalid one was encountered.
+
+  Returns:
+      test.integration.common.IpVersion: Array with ip versions obtained from the ENVOY_IP_TEST_VERSIONS environment variable.
```

Comment:
(nit) Return values are typically formatted as a set of sentences, without the "type:", e.g:

```py
Returns:
  A list of test.integration.common.IpVersion with ip versions obtained from the 
  ENVOY_IP_TEST_VERSIONS environment variable.
```

Applies in a few places.